dreamplace/ops/density_potential/src/density_potential_cuda.cpp

```diff
@@ -172,7 +172,7 @@ std::vector<at::Tensor> density_potential_forward(
     // set padding density 
     if (padding > 0)
     {
-        density_map.masked_fill_(padding_mask, at::Scalar(target_area));
+        density_map.masked_fill_(padding_mask.to(at::ScalarType::Bool), at::Scalar(target_area));
```

Comment:
Can you find equivalent python API for ```padding_mask.to(at::ScalarType::Bool)```? I am more comfortable to do it in python.

Reply:
I have moved it to python.